ports/nrf/boards/supermini_nrf52840/mpconfigboard.mk

```diff
@@ -0,0 +1,8 @@
+USB_VID = 0x239A
+USB_PID = 0x8029
```

Comment:
Awaiting update to approved pid.codes VID/PID.

Reply:
pid.codes PR is merged. Please update the values here.

Reply:
requested changes implemented.